documentation/bibliography.bib

```diff
@@ -956,3 +956,19 @@ @unpublished{Celledoni2007
   title  = {The exact computation of the free rigid body motion and its use in splitting methods},
 }
 
+@book{Brent1973,
+  author    = {Brent, Richard P.},
+  publisher = {Prentice-Hall},
+  date      = {1973},
+  title     = {Algorithms for minimization without derivatives},
+}
+
+@article{WilkinsGu2012,
+  author       = {Wilkins, Gautam and Gu, Ming},
+  date         = {2012},
+  doi          = {10.1007/s00211-012-0480-x},
+  journaltitle = {Numerische Mathematik},
+  number       = {123},
+  pages        = {177--188},
+  title        = {A modified {B}rent’s method for finding zeros of functions},
+}
```

Comment:
Please keep this file properly formatted: the stanzas must be ordered by type (`@article` before `@book`) then identifier (`Brent1973` before `ByrdFriedman1954`) and the attributes must be sorted. (`date` before `publisher`).  In other words, follow the style of the code around you.

These additions should go near lines 766 and 742, respectively.